python/src/addons/ForwardBackwardCompletion.pyx

```diff
@@ -0,0 +1,4 @@
+
+
+    def GetCompletions(self, *args):
+        return call_deprecated_method("GetCompletions", "complete", self.complete, *args)
```

Comment:
isn't `get_completions` better ?

Reply:
:thinking: 

In a previous change I named it like [this](https://keyvidev.github.io/keyvi/python/dictionary.html):

- complete_prefix
- complete_multiword
- complete_fuzzy

avoiding prefixes like `get_`

Reply:
Ah, okie ...